### DISCUSSION TITLE
pyannote.audio 2.0.1 document

### DISCUSSION THREAD

**Discussion by mehdibakhtiyarifar01** [pyannote/pyannote-audio#1080] [Q&A]
Hi i am a student and I am working on the pyannote-audio 2.0.1 for research work.
I want to know do you have any documentation for the pyannote-audio 2.0.1?
@hbredin

**Comment by hbredin** [comment]
Partial but yes. https://github.com/pyannote/pyannote-audio#documentation

Feel free to contribute more.
